content/docs/tutorials/getting-started-aws-letsencrypt/README.md

```diff
@@ -12,8 +12,8 @@ In this tutorial you will learn how to deploy and configure cert-manager on AWS
 and how to deploy an HTTPS web server and make it available on the Internet.
 You will learn how to configure cert-manager to get a signed certificate from Let's Encrypt,
 which will allow clients to connect to your HTTPS website securely.
-You will configure cert-manager to use the [Let's Encrypt DNS-01 challenge protocol](https://letsencrypt.org/docs/challenge-types/#dns-01-challenge) with AWS Route53 DNS,
-using IAM Roles for Service Accounts (IRSA) to authenticate to AWS.
+You will configure cert-manager to use the [Let's Encrypt DNS-01 challenge protocol](https://letsencrypt.org/docs/challenge-types/#dns-01-challenge) with AWS Route53 DNS.
+You will authenticate to Route53 using a [dedicated Kubernetes ServiceAccount token](../../configuration/acme/dns01/route53.md#referencing-your-own-serviceaccount-within-in-an-issuer-or-clusterissuer).
```

Comment:
At the time I wrote the tutorial, I thought that IRSA referred to any auth mechanism that used a Kubernetes ServiceAccount token.
I now think that IRSA more of a marketing product name which comprises annotated Kubernetes ServiceAccount + mutating webhook (to inject a projected service account volume) +  Kubernetes ServiceAccount token (signed JWT) loaded from `/var/run/aws/token`.

This tutorial does not use the webhook or projected service account token volume.

In future it might be worth updating the tutorial to use the much simpler "Pod Identity" mechanism,
because that will probably be more familiar to AWS EKS users.